view/adminhtml/templates/menu/nodes.phtml

```diff
@@ -46,6 +46,7 @@ $blockHtmlId = $block->getBlockHtmlId();
                         "imageDeleteUrl"   : "<?= $block->getImageDeleteUrl() ?>",
                         "imageUploadFileId": "<?= $block->getImageUploadFileId() ?>",
                         "lazyMinItemsCount": "1000",
+                        "serializedNodesElementName": "serialized_nodes",
```

Comment:
TODO: pass the correct value from backend (either "serialized_nodes" or "snowmenu_staging_serialized_nodes")